### PR TITLE
Issue 127: Added Google Anaytics Events tracking script to BORG theme. Issue 153: Search Identifier should be uniform. 

### PR DESCRIPTION
**Pull request by thejimbirch**
Closely watch google-analytics tracking #127 

Google Analytics tracks events in their javascript on successful loading of the javascript on the target page.  Since some of the events are offsite (Zip File, Pantheon, Mailchimp), we need to trigger the event, then send the user to the goal.

Search Identifier should be uniform #153 

Updated Search Identifiers on the Exposed Filters of the 3 displays of the modules view to be s in both views.view.modules.json in the repo.

### CODE REVIEW

File: config/dev-active/views.view.modules.json [hunk 1]
```diff
@@ -455,7 +455,7 @@
                             "description": "",
                             "use_operator": 0,
                             "operator": "combine_op",
-                            "identifier": "combine",
+                            "identifier": "s",
```
Comment (jenlampton) [inline config/dev-active/views.view.modules.json]: Can we use an identifier that's a little more descriptive than 's'?  How about 'search'? :)

**Comment by jenlampton** [comment]
This is looking really good. I can merge whenever, just had the one Q about `s`.

**Comment by thejimbirch** [comment]
Hi @jenlampton 

I copied the `s` from Wordpress.  Their default search is:
http://www.catsinmyyard.com/?s=Chicago

Google uses `q`:
https://www.google.com/?q=Backdrop

But I didn't think that was a good idea since Backdrop core uses `q` when "clean urls" are not enabled:
example.com/?q=user

If you would like me to change, let me know.

**Comment by jenlampton** [comment]
I like copying wordpress :)

**Comment by jenlampton** [comment]
In general, for PRs against backdropcms.org we shouldn't change both copies of the config file. We should change only the ones in the dev-active directory, since we'll (usually) need to run the import script on live after pulling code. The import script on the live environment will pull the changed config files from dev-active, so that's where the changes need to be.

For development, my process is to git pull the change, then manually copy the new config files from dev-active into live-active and run the import on my dev site, to be sure the import works successfully. It's a bit of a hassle but it works!

In this case (since the update script doesn't change anything) we can get away with a cache-clear instead, so I merged the PR. deploying now :)

**Comment by thejimbirch** [comment]
Thanks for the explanation @jenlampton I am a bit behind with my comprehension of the configuration system.

**Comment by jenlampton** [comment]
NP This is a particularly trixy problem, too.

**Pull request merged**